pkg/errors/errors.go

```diff
@@ -1,6 +1,9 @@
 package v3ioerrors
```

Comment:
Hmm, this package should actually be removed in favor of nuclio/errors

Reply:
Sure? Assuming not in this PR. It will be a breaking change when we do it.